packages/tables/src/views/Table.js

```diff
@@ -61,6 +73,10 @@ const Table = styled.table.attrs({
 })`
   ${props => retrieveSrollableStyling(props)};
   ${props => retrieveTheme(COMPONENT_ID, props)};
+
+  ${/* sc-selector */ StyledRow} {
```

Comment:
This specifically overrides the `height` and `min-height` properties for the row. Since we are no longer able to rely on the `display: table` styling to re-flow the row heights, we have to modify the styling.

Since the default `Row` export isn't a styled component, the `StyledRow` is an additional export that allows us to style against it.

Reply:
Also, `/* sc-selector */` is a "fix" for stylelint being confused by the template literal https://www.styled-components.com/docs/tooling#tags